apps/frontend/app/communitiesUser/componets/CommunityCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { Community } from "@/app/utils/Connections/connectionsCommunity";
+import React from "react";
+import Image from 'next/image';
+
+type Props = {
+  community: Community;
+  onClick: () => void;
+  buttonText?: string; // Propiedad opcional para el texto del botón
```

Comment:
If there are going to be comments in the code, please make them in English and on a new line.